dashboard/app/views/scripts/show.html.haml

```diff
@@ -75,3 +76,7 @@
         %ul
           - @script.unit_groups.each do |course|
             %li= link_to course_path(course)
+      -if lesson_plans && lesson_plans.count > 0
```

Comment:
I worry about this making the admin box feel cluttered. What was the reason for adding this?

Reply:
because now the View Lesson Plan links do not take you to the lesson plans on code studio, so I thought it would be better to make it easier to get to them. otherwise, it's script overview --> script edit --> lesson edit --> lesson plan.

one idea would be to only show this if there is actually lesson plan content in the process of being reviewed, e.g. `use_legacy_lesson_plans` and also has code studio lesson plan content. since I'm not sure what the "has code studio lesson plan content" condition will be yet, I'll just limit this for now to lesson plans with `use_legacy_lesson_plans` enabled.

Reply:
adding a note to remember to do this here: https://codedotorg.atlassian.net/browse/PLAT-830

Reply:
This plan sounds great to me.